packages/frontend/core/src/modules/docs-search/worker/types.ts

```diff
@@ -34,7 +34,7 @@ export type WorkerInput =
     }
   | {
       type: 'doc';
-      docId: string;
+      storageDocId: string;
```

Comment:
can you explain changes to `storageDocId`? to increase worker's inefficiency?

Reply:
This is due to historical reasons. The id of the doc stored may not be consistent with the docid. The current solution is to traverse the space map in the rootDoc ​ and query the docid from the storageDocId in the worker.

Reply:
This is to fix the issue that some old docs not automatically update the index

Reply:
I see, thanks!